requirements.txt

```diff
@@ -5,3 +5,8 @@
 pytest>=3.2.2
 sphinx>=1.6.4
 sphinx_rtd_theme>=0.2.5b1
+backports.tempfile>=1.0rc1
```

Comment:
I was able to install, test, and also build the docs even without these additions to requirements.txt